test/tests/mesh/checkpoint/tests

```diff
@@ -1,56 +1,59 @@
 [Tests]
   [./test_2]
+    # To generate the splits for this test, run
+    # moose_test-opt -i checkpoint_split.i --split-mesh 2 --split-file checkpoint_split_in
     type = 'Exodiff'
     input = 'checkpoint_split.i'
     exodiff = 'test_2.e'
-    cli_args = 'Outputs/file_base=test_2  Mesh/file=checkpoint_split_in.2.cpr'
-    max_parallel = 2  # workaround until libMesh/libmesh#1408
-    min_parallel = 2  # workaround until libMesh/libmesh#1409
-    skip = 'Requires libMesh/libmesh#1409'
+    cli_args = 'Outputs/file_base=test_2 --use-split --split-file checkpoint_split_in'
+    max_parallel = 2
+    min_parallel = 2
   [../]
   [./test_2a]
     type = 'Exodiff'
     input = 'checkpoint_split.i'
     exodiff = 'test_2a.e'
-    cli_args = 'Outputs/file_base=test_2a Mesh/file=checkpoint_split_in.2.cpa'
-    max_parallel = 2  # workaround until libMesh/libmesh#1408
-    min_parallel = 2  # workaround until libMesh/libmesh#1409
-    skip = 'Requires libMesh/libmesh#1409'
+    cli_args = 'Outputs/file_base=test_2a --use-split --split-file checkpoint_split_in --split-ascii'
```

Comment:
We should just update MOOSE to switch formats based on the ``--split-file`` extension (e.g. cpa vs cpr)  and assume cpr if no extension is present or ``--split-file`` isn't specified.